crates/turborepo-globwalk/src/lib.rs

```diff
@@ -308,82 +307,60 @@ pub fn globwalk_internal(
 ) -> Result<HashSet<AbsoluteSystemPathBuf>, WalkError> {
     let (base_path_new, include_paths, exclude_paths) =
         preprocess_paths_and_globs(base_path, include, exclude)?;
-    let inc_patterns = include_paths
-        .iter()
-        .map(glob_with_contextual_error)
-        .collect::<Result<Vec<_>, WalkError>>()?;
+
     let ex_patterns = exclude_paths
-        .iter()
+        .into_iter()
         .map(glob_with_contextual_error)
-        .collect::<Result<Vec<_>, _>>()?;
+        .collect::<Result<_, _>>()?;
 
-    let span = Span::current();
-    let result = inc_patterns
+    include_paths
         .into_iter()
-        .flat_map(|glob| {
```

Comment:
Are the checks performed by wax's `glob.walk` now?

Reply:
PR is ready to review, I added some explanations in the body